src/webhook/validation/deprecation.go

```diff
@@ -65,11 +65,22 @@ func deprecatedFeatureFlagDisableMetadataEnrichment(_ *dynakubeValidator, dynaku
 }
 
 func deprecatedFeatureFlag(_ *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
-	warnings := ""
+	warningMessage := ""
 	for _, ff := range getDeprecatedFeatureFlags() {
-		warnings += checkDeprecatedFeatureFlag(dynakube, ff)
+		if checkDeprecatedFeatureFlag(dynakube, ff) {
+			warningMessage += fmt.Sprintf("'%s'", ff) + " "
```

Comment:
you can collect the feature flags in an array, and then combine them at the end using `strings.join(ffs, ", ")`